src/undate/converters/calendars/hijri/converter.py

```diff
@@ -24,10 +24,6 @@ class HijriDateConverter(BaseCalendarConverter):
     def __init__(self):
         self.transformer = HijriDateTransformer()
 
-    def max_month(self, year: int) -> int:
-        """maximum numeric month for the specified year in this calendar"""
-        return 12
-
```

Comment:
This is being pulled because the base converter now defaults to `12`?

Reply:
I think I prefer keeping the unimplemented methods as stubs in the base and requiring specific converters to implement them, since there aren't too many. But it makes sense either way